src/screens/SignIn/form.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import FormButton from 'components/FormButton'
+
+type SignInFormProps = {
+  submitHandler?: React.FormEventHandler
```

Comment:
That's a good idea. Fixed in d0b9f800bf4ec475cf4b55d26ab6f238cf16fcb7
I made it optional because I don't want to send this from the test but it make more sense to be required.